Remotion/Core/Collections.DataStore.UnitTests/ExpiringDataStoreTest.cs

```diff
@@ -41,10 +42,10 @@ public class ExpiringDataStoreTest
     public void SetUp ()
     {
       _initialScanInfo = new DateTime (5);
-      _expirationPolicyMock = MockRepository.GenerateStrictMock<IExpirationPolicy<object, DateTime, DateTime>> ();
-      _expirationPolicyMock.Stub (stub => stub.GetNextScanInfo()).Return (_initialScanInfo).Repeat.Once(); // just for the ctor
```

Comment:
in Moq `Repeat.Once` would be checked with `Verify (..., Times.Once)`, but since this is a stub there is no Verify and therefore the `Repeat.Once` is not checked

Reply:
Problem: RhinoMocks still goes boom! Combinations of Strict-Mock + Repeat.Once + Stub need special handling, too.

Same goes for just Stub + Repeat.Once since this means only the first retunr value is set up, later once return null.

Reply:
What is the best way to handle such special cases?